src/plugin/ipc/file/fileconnlist.cpp

```diff
@@ -473,30 +473,55 @@ FileConnList::recreateShmFileAndMap(const ProcMapsArea &area)
   JASSERT(createDirectoryTree(area.name)) (area.name)
   .Text("Unable to create directory in File Path");
 
```

Comment:
I suggest an added comment on the next line:
```
// NOTE:  Unlinking /libhugetlbfs.tmp is different from unlinking an ordinary file.
//             The file is a virtual file.  It is not really unlinked, and the
//             memory segment was originally backed by this solely to force
//             the use of hugepages.
```
Feel free to re-phrase anything if you think it's not clear.